docs/pages/repo/docs/core-concepts/caching/what-to-cache.mdx

```diff
@@ -0,0 +1,85 @@
+---
+title: What to Cache?
+description: The configuration of Turborepo `outputs`
+---
+
+import Callout from "../../../../../components/Callout";
+
+# What to Cache?
+
+By default, the console output (also known as `stdout`) from all tasks is _always_ captured by Turborepo. If your task does not emit any files (e.g. unit tests with Jest) you can omit `outputs`. Beyond that, you will generally need to specify which files on disk you need to capture (and restore!) in order to be able to run subsequent dependent tasks.
```

Comment:
As above, we cache `stdout` _and_ `stderr`. You can probably just drop the part in parenthesis. "console output" covers both.

Reply:
Yeah, this paragraph needs some love. It's mostly 1:1 with the existing state.